### DISCUSSION TITLE
MCX/NeuroJSON joint in-person training workshop 2024

### DISCUSSION THREAD

**Discussion by leoyala** [fangq/mcx#218] [General]
Hello. I just received an email about the in-person workshop. Is it also possible to attend remotely? I know that this would be relevant for several people in my team. However, they would not be able to join in-person. Or would the sessions be recorded and made available online?

**Comment by fangq** [comment]
hi @leoyala, sorry for the late reply, was traveling. 

It is possible, although my concern is that when we go over tools such as MCX Cloud, where only limited GPU are available, I worry that our hardware won't be able to support all online users and in-person participants simultaneously. We also unlikely have the bandwidth to help all online users to trouble-shoot problems while running the in-person workshop.

However, I am considering the option that users who are familiar with mcx with a working local setup could use their own laptop/servers to participate our workshop remotely. Again, we may not have manpower to help troubleshoot during the workshop, but you can potentially "observe" all of our training materials.

let me figure out the logistics and hopefully will announce the support/requirement in the mailing list.
